src/pages/Dashboard/components/ReportTable/TableRow/TableRow.tsx

```diff
@@ -84,7 +89,9 @@ function TableRow({ skill, projections }: TableRowProps): JSX.Element {
     <Tbody bg="white" w="100%" borderRadius="lg">
       <Tr>
         <Th
-          rowSpan={isExpanded ? 3 + maxNeededRoles + maxBenchEmployees : 3}
+          rowSpan={
+            isExpanded ? 3 + maxNeededRolesExpandable + maxBenchEmployees : 3
```

Comment:
Looks like this `3` was here before this PR but, Is `3` the min row span? It might be helpful to have it pulled out to a constant, same with the `Td` below